active_utils.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
 
   s.rubyforge_project = "active_utils"
 
-  s.add_dependency('activesupport', '>= 3.2')
+  s.add_dependency('activesupport', '>= 3.2', '< 5.1.0')
```

Comment:
Not against this, but this forces us to do a release once 5.1 gets released. If we don't do that right away, you probably get people complaining about this. This may be fine, but be aware :)